packages/main/src/themes/DayPicker.css

```diff
@@ -11,17 +11,17 @@
 	flex-basis: 100%;
 }
 
-:host([secondary-calendar-type]) .ui5-dp-item {
+.ui5-dp-root.ui5-dp-twocalendartypes .ui5-dp-item {
```

Comment:
do we need .ui5-dp-root.ui5-dp-twocalendartypes instead of .ui5-dp-twocalendartypes

Reply:
I just use it to ensure stronger selector. To the question, both selectors will work. I just use it for stroger selection.